scripts/management/slurm_query_status.py

```diff
@@ -84,10 +84,11 @@ def is_task_complete(task, task_list):
 def check_dependancy_met(task, task_list):
     process, run_name, status = task
     process = db_helper.Process(process)
-    if process is Process.EMOD3D or process is Process.HF:
+    if process in (Process.EMOD3D, Process.HF, Process.rrup):
```

Comment:
Should Process.Verification be in one of these if statements?

Reply:
Yupp, but as you said currently it does nothing for now.